src/coreclr/hosts/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 include_directories(inc)
+include_directories(../../inc)
```

Comment:
I was wondering why we need this added include and so I've tried to compile without this change - and it was ok both on Linux and Windows.